### DISCUSSION TITLE
존재하지 않는 음악 검색시 사용자에게 표시되는 문구 #99

### DISCUSSION THREAD

**Discussion by YunSeok-Choi** [DeveloperAcademy-POSTECH/MC2-Team7-Larasy#104] [Q&A] [answered]
> - #99

음악이 존재하지 않는다면 로딩이 멈추고 난 후에 아무런 표시 없이 공백으로 주어 줄지, 존재하지 않는 음악이라는 문구를 준다면 어떤 표현이 좋을까요?

- ` "해당 음악은 존재하지 않습니다." `

- ` "노래의 제목을 다시 한번 확인해주세요." `

- ` "음악이 검색되지 않았습니다." `

등등 좋은 문구가 있다면 코멘트 달아주세요! 
해당 문구로 표기하겠습니다!

- 만약 문구가 딱히 필요한 것같지 않다고 생각하시면 아래 🚀 이모지를 눌러주세요!

**Comment by JIWON1923** [comment, marked as answer]
<img src = "https://user-images.githubusercontent.com/68676844/225818088-6562bc25-ebfb-4684-9460-7a43168a1d3b.PNG" width=250>

이런 느낌은 어떨까요? (멜론의 화면입니다!)

"\(검색어)를 찾을 수 없어요"
철자와 띄어쓰기 어쩌고.. 확인해주세요

**Comment by JMM00** [comment]
검색 결과가 없는데 아무런 문구가 나타나지 않는다면 사용자가 오류라고 생각할 수도 있어 문구를 넣는 것이 좋다고 생각합니다. 다만 실제로 없는 음악을 검색하기보다는 잘못된 제목으로 검색할 가능성이 더 높다고 생각되어 음악이 존재하지 않는다는 문구보다는 검색 결과가 존재하지 않는다는 문구가 더 적절하다고 생각합니다. 
이런 맥락에서 지원님이 올려주신 '검색결과가 없습니다.' 가 가장 괜찮다고 생각합니다!

**Comment by YunSeok-Choi** [comment]
모두 의견 주셔서 감사합니다! 두분의 의견대로
검색 결과가 없습니다. 로 표기하거나 비슷한 뉘앙스로 표기해보겠습니다!
